composer.json

```diff
@@ -16,17 +16,18 @@
         }
     },
     "require": {
-        "php": "^7.1 || ^8.0",
+        "php": "^7.4 || ^8.0",
```

Comment:
This would also have to be reflected in the CI configuration - currently jobs with PHP<7.4 fail because composer can't resolve the dependencies.

Reply:
Oh sorry, haven't realized that!